src/lib/components/SpiderChart.svelte

```diff
@@ -157,7 +188,17 @@
 				{#if innerWidth >= BREAKPOINT}
 					{label.text}
 				{:else}
-					Dynamic {idx + 1}
+					<div
+						onmouseover={() => mouseOver(dynamics[idx])}
+						onmousemove={mouseMove}
+						onmouseout={() => removeDescription(tooltip)}
+						onfocus={() => focus(dynamics[idx])}
+						onblur={() => removeDescription(hidden)}
+						role="button"
+						tabindex={idx + 2}
```

Comment:
a bit hacky - I added 2 here to ensure that Dynamic 1 was the first one that was focused on when tabbing through the page